src/Front/Check.hs

```diff
@@ -1,46 +0,0 @@
-module Front.Check where
```

Comment:
Deleted this since it's something that's not really consistent with the presence of type inference.

We can bring this back once we add warning infrastructure.